src/Raven.Client/Documents/Session/IDocumentQueryBase.cs

```diff
@@ -434,39 +434,35 @@ public interface IGroupByDocumentQueryBase<T, TSelf> where TSelf : IDocumentQuer
     }
 
     /// <summary>
-    ///     A query against a Raven index
+    ///     Interface for lower level API access.
```

Comment:
Interface for lower level API access. -> Interface providing low-level querying capabilities

Reply:
Changed